x/evm/genesis.go

```diff
@@ -83,6 +90,62 @@ func InitGenesis(
 		}
 	}
 
+	if utils.IsEthermintDevChain(ctx) {
```

Comment:
why we care about ethermint devnet?

Reply:
This is the development flow. I develop the feature for Ethermint individually so the feature can be brought to any chains those use this fork-ethermint. That why the feature must be available on Ethermint Devnet as well.

Reply:
One more point I forgot to mention: for test set, on this Ethermint.